LegitimateURLShortener.txt

```diff
@@ -3152,6 +3157,9 @@ $removeparam=_amp,domain=latimes.com
 ! https://metro.co.uk/2022/02/07/facebook-and-insta-may-drop-europe-if-they-cant-target-users-with-ads-16062662/?ico=related-posts
 $removeparam=ico,domain=metro.co.uk
 
+! https://www.disneyplus.com/en-fi?cid=DSS-Affiliate-Impact-Content-Future%20PLC.-564546
+$doc,removeparam=/^cid=DSS-Affiliate/,domain=disneyplus.com
```

Comment:
I am not sure if it still is an issue, but I though that $doc + $removeparam caused issues in AdGuard. @DandelionSprout

Reply:
Well if it is an issue, there's also some other $doc,removeparam filters in this list.

Reply:
Removed it for now.

Reply:
Like I said, I'm not sure, I just remember Imre saying something, but that was a while ago (there were some other $doc-related issues in AdGuard but those were patched)